README.md

```diff
@@ -1,38 +1,135 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Polis
 
-## Getting Started
+---
 
-First, run the development server:
+Polis is a platform dedicated to showcasing web3 tools, templates, and boilerplates. Frequently, these resources remain isolated or challenging to find within the web3 landscape. Polis is committed to addressing this challenge by offering a solution to enhance their discoverability and accessibility.
 
-```bash
+---
+
+## How It Works
+
+![polis](./public/polis.png)
+
+**If you are a Contributor:**
+
+- You can submit new tools/boilerplates/templates
+- You will also be able to edit the application later
+
+Note: Contributors will need to connect their wallet to be able to submit and edit the applications.
+
+**If you are a User:**
+
+- You can explore tools, templates and boilerplates on Polis
+- You will find all necessary web3 tools/templates/boilerplates in one place
+
+**What are the main features?**
+
+- Connect Wallet
+- Submit New Applications: _Can contribute and share your own application on the platform_
+- Edit Applications: _Contributors are able to edit the applications they've submitted_
+- Editor's Pick: _Users can discover the applications most used by the community_
+
+## Technical Details:
+
+| Name   | Link                            | Text           |
+| ------ | ------------------------------- | -------------- |
+| Nodejs | https://nodejs.org/en/download/ | Latest Version |
+| Infura | https://infura.io/              |                |
+
+### **Libraries used**
+
+Here's an overview of the included frameworks and tools.
+
+- **Next.js** - A frameworks fo React applications that is focused on server-rendered and minimalistic approach.
+- **Typescript** - Superset of JavaScript which primarily provides optional static typing, classes and interfaces.
+- **Tailwind CSS**- UI & Styling Library.
+- **[Web3 Storage](https://web3.storage/)**- A decentralized storage solution which natively uses decentralized data and identity protocols like IPFS, Filecoin and UCAN.
+
+---
+
+## Run It Locally
+
+1. Clone the repository
+
+```
+git@github.com:Consensys/polis.git
+```
+
+2. Install Dependancies
+
+```
+cd polis
+npm install
+```
+
+3. You will need to create a new `.env` file following`.env.example` file. Your `.env` file should look like this:
+
+```
+INFURA_API_KEY=
+WEB3_NAME_SERVICE_URL=https://name.web3.storage/name
```

Comment:
maybe an information on how to get that value here? `https://name.web3.storage/name`

Reply:
@meowyx  Here's the doc to support that 
https://web3.storage/docs/reference/w3name-http-api/